seq2science/scripts/generate_linked_txome.R

```diff
@@ -40,17 +35,20 @@ cat('\n')
 ## create a symlink to the gtf with Ensembl naming scheme (required for tximeta)
 fake_gtf_path <- file.path(dirname(output), paste0(organism, '.', genome, '.', release, '.gtf'))
```

Comment:
and in here?

Reply:
I ran into issues only during the tests in this PR

Reply:
as there were merge conflicts relating to tximeta, i guess the main branch caught up with whatever was causing them. I've combined the fixes (I think).